flake.nix

```diff
@@ -0,0 +1,55 @@
+{
+  inputs = {
+    nixpkgs.url = "github:NixOS/nixpkgs/nixos-22.05";
+    utils.url = "github:gytis-ivaskevicius/flake-utils-plus";
+  };
+
+  outputs = inputs@{ self, utils, ... }:
+    utils.lib.mkFlake rec {
+      inherit self inputs;
+
+      supportedSystems = [
+        "aarch64-linux"
+        "aarch64-darwin"
+        "i686-linux"
+        "x86_64-darwin"
+        "x86_64-linux"
+      ];
+
+      outputsBuilder = channels: with channels; {
+        packages = with nixpkgs; { 
+          inherit (nixpkgs) package-from-overlays;
+
+          nixpacks = rustPlatform.buildRustPackage rec {
+            pname = "nixpacks";
+            version = "v0.0.20";
+            doCheck = true;
+            src = ./.;
+            nativeBuildInputs = [ clippy ];
```

Comment:
Shouldn't rustfmt be here?

Reply:
I don't think so... In `checkPhase` I execute commands following CONTRIBUTING to mimic a "local CI" before build applicaiton and `rustfmt` isn't executed in this flow. I added `clippy` because is a external command oppositely to `cargo check` and `cargo test`.

`nativeBuildInputs` add extra packages to be available only at build time!

Reply:
added